test/Governance.ts

```diff
@@ -1324,6 +1324,65 @@ describe("Governance test", function () {
     });
 });
 
+describe.only("Proposals test", function () {
+    beforeEach(async function (){
+        Object.assign(this, await loadFixture(governanceFixture));
+    });
+    it("SoftwareUpgradeProposal", async function () {
+        const upgradableProxy: UpgradableCounterContract = await upgrades.deployProxy(
+            await ethers.getContractFactory('UpgradableCounterContract'),
+            [await this.gov.getAddress()],
+            {kind: 'uups'}
+        );
+        await this.verifier.addTemplate(
+            3,
```

Comment:
please add comment, why the template, like:
```
// register UNKNOWN_DELEGATECALL_EXECUTABLE proposal type (pType used by SoftwareUpgradeProposal)
```
(this smells really bad in the governance codebase, lets discuss it out of this PR)